lib/Agrammon/Web/Routes.pm6

```diff
@@ -165,11 +165,11 @@ sub dataset-routes(Agrammon::Web::Service $ws) {
             }
         }
 
-        # test
+        # working
         post -> LoggedIn $user, 'rename_tag' {
-            request-body -> (:$old!, :$new!) {
-                my $data = $ws.rename-tag($user, $old, $new);
-                content 'application/json', $data;
+            request-body -> (:oldTag($old)!, :newTag($new)!) {
+                my $new-name = $ws.rename-tag($user, $old, $new);
+                content 'application/json', %( :newName($new-name) );
```

Comment:
Also a question of what we do if the name didn't exist, or was in conflict with the new one.

Reply:
Added error handling and OpenAPI validation